README.md

```diff
@@ -10,3 +10,73 @@ addon framework.
 
 * [Open Cluster Management](https://github.com/open-cluster-management-io/)
 * [OpenShift Data Foundation Operator](https://github.com/red-hat-storage/odf-operator)
+
+## Setting up operator locally for development
+
+### Pre-requisites:
+a. Have kubectl, minikube, docker/podman installed.  
+b. Login to the relevant container registry (eg - docker, quay) 
+
+### Setup minikube with OLM 
+> minikube start && minikube addons enable olm
+
+### Building the operator 
+> export REGISTRY_NAMESPACE={YOUR_REGISTRY_NAMESPACE}
+> export IMAGE_TAG={YOUR_IMAGE_TAG}
+> make generate  
+> make manifests  
+> make build  
+> make docker-build  
+> make docker-push  
+> make bundle  
+> make bundle-build  
+> make bundle-push  
+> make catalog-build  
+> make catalog-push  
+
+### Edit the CatalogSource, Subscription and apply to cluster 
```

Comment:
Create a separate section for local deployment and move these under that section.

Reply:
also, add kubernetes cluster with OLM installed as pre-req instead of minikube.